src/constants.ts

```diff
@@ -20,7 +20,7 @@ export const GETH_DISCORD_URL = 'https://discord.com/invite/nthXNEv';
 export const GO_URL = 'https://go.dev/';
 
 // Downloads
-export const DEFAULT_BUILD_AMOUNT_TO_SHOW = 10;
+export const DEFAULT_BUILD_AMOUNT_TO_SHOW = 12;
```

Comment:
@nhsz Does this fix the issue posted (#78)?

> we wanted to always display the last 2 versions - independent of how many lines that is - and if it's not yet 10, keep pulling more in - we should retain that logic

It doesn't sound like we wanted to change the "more" from 10 to 12... it sounded like we wanted to still use 10, but also check how many are needed to fully display the latest two releases for a given OS, and make sure those are always displayed.

Reply:
this was discussed on the last sync w/ @jmcook1186, not very clear on the issue description but what they wanted was to always show the latest 2 releases for linux and this means showing 12 rows as each linux release equals to 6 build artifacts, 2 releases = 12. It's not relevant for the other OSes, just for linux

Reply:
Ok, works for me if we're comfortable just hard coding that as 12... didn't know if that would be something that may change over time